lib/fathead/python_requests/output.txt

```diff
@@ -1,36 +1,36 @@
+Body Content Workflow	A										<p>By default, when you make a request, the body of the response is downloaded \nimmediately. You can override this behaviour and defer downloading the response \nbody until you access the Response.content \nattribute with the stream parameter:<pre>b"tarball_url = 'https://github.com/kennethreitz/requests/tarball/master'\\\\nr = requests.get(tarball_url, stream=True)\\\\n"</pre> At this point only the response headers have been downloaded and the connection \nremains open, hence allowing us to make content retrieval conditional:<pre>b"if int(r.headers['content-length']) < TOO_LONG:\\\\n  content = r.content\\\\n  ...\\\\n"</pre> You can further control the workflow by use of the Response.iter_content() \nand Response.iter_lines() methods. \nAlternatively, you can read the undecoded body from the underlying \nurllib3 urllib3.HTTPResponse at \nResponse.raw. If you set stream to True when making a request, Requests cannot \nrelease the connection back to the pool unless you consume all the data or call \nResponse.close. This can lead to \ninefficiency with connections. If you find yourself partially reading request \nbodies (or not reading them at all) while using stream=True, you should \nconsider using contextlib.closing (documented here), like this:<pre>b"from contextlib import closing\\\\n\\\\nwith closing(requests.get('http://httpbin.org/get', stream=True)) as r:\\\\n    # Do things with the response here.\\\\n"</pre></p>	http://docs.python-requests.org/en/master/user/advanced/#body-content-workflow
```

Comment:
@hchienjo looks like we're missing some ` \n` here. Causing the output to break still:

https://beta.duckduckgo.com/?q=Body+Content+Workflow&ia=about&iax=1

Notice the missing space between "downloaded" and "immediately"